src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -6,12 +6,14 @@
 import java.time.LocalDateTime;
 import java.util.Objects;
 
+import seedu.contax.model.chrono.ScheduleItem;
 import seedu.contax.model.person.Person;
 
 /**
- * Represents an appointment in the schedule.
+ * Represents an appointment in the Schedule. Time related functionality is implemented in the superclass
```

Comment:
```suggestion
 * Represents an appointment in the Schedule. Time related functionality is implemented in the superclass.
```

Reply:
Thanks for spotting it, fixed in 827b102.